### PR TITLE
Dedicated mongo

### PR DESCRIPTION
**Pull request by frosario**

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -83,12 +83,12 @@ girder_item_template_source:
               type: string
               index: not_analyzed
 
-mongo_url: localhost
-celery_broker: "mongodb://localhost:27017/tasks"
+mongo_url: 172.30.2.123
```
Comment (nathanathan) [inline group_vars/all.yml]: I recommend overriding the localhost urls with specific ips in the docker.yml or prod.yml file.

File: roles/girder/tasks/main.yml [hunk 2]
```diff
@@ -90,13 +90,10 @@
...
 
-  - name: Install grunt globally
-    npm: name={{ item }} global=yes
-    with_items:
-      - grunt
-      - grunt-cli
-    sudo: yes
-    sudo_user: root
+  - name: "Ensure grunt is installed"
+    command: npm -g install grunt grunt-cli || true
```
Comment (nathanathan) [inline roles/girder/tasks/main.yml]: What is the || true used for?

File: group_vars/all.yml (hunk 1 above)
Comment (frosario) [inline group_vars/all.yml]: Followed recommendation here:
https://github.com/ecohealthalliance/grits-deploy-ansible/commit/cc02ee9c125c2cbc3ff038caffe1c2bfdd6a025a

File: roles/girder/tasks/main.yml (hunk 2 above)
Comment (frosario) [inline roles/girder/tasks/main.yml]: That was something I forgot to remove. Fixed that with this commit:
https://github.com/ecohealthalliance/grits-deploy-ansible/commit/6c752eb89a49be3ff6d03627119c0249432831ed

**Pull request merged**